dotCMS/src/main/java/com/dotcms/experiments/business/web/ExperimentWebAPIImpl.java

```diff
@@ -147,10 +161,14 @@ private SelectedExperiment getExperimentSelected(final Experiment experiment,
         }
     }
 
-    private List<Experiment> pickExperiments(final List<Experiment> runningExperiments,
+    private Optional<List<Experiment>> pickExperiments(final List<Experiment> runningExperiments,
```

Comment:
Why do you need an Optional List instead of returning empty list?

Reply:
done https://github.com/dotCMS/core/pull/27989/commits/b1d927cd36b8975d11b2c4b0e92e3ee543e41133#diff-df4c3ee11b473e0f4e91d5be7a6107bf6cdb4ab40bf0661a182dd298e00e5602R170